x/concentrated-liquidity/grpc_query.go

```diff
@@ -77,8 +78,37 @@ func (q Querier) UserPositions(ctx context.Context, req *clquery.QueryUserPositi
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	var positions []model.PositionWithUnderlyingAssetBreakdown
```

Comment:
Please cover this logic with some tests

Reply:
Please lmk if this is sufficient! https://github.com/osmosis-labs/osmosis/pull/4637/commits/c3290f73de27ad84ca8bcc04d5de5072fa122628